### ISSUE TITLE
[bug] broken on mac os

### ISSUE THREAD

**Issue by justchokingaround** [jstkdng/ueberzugpp#120]
i can't get uz++ to display anything after installing it with the brew formula. i tried running `export SPDLOG_LEVEL=debug` before calling uz++ but there was no log for it in `/tmp/`. i'm not sure how else to debug

**Comment by jstkdng** [comment]
on macos, only the iterm2, kitty and sixel backends are available. X11 is disabled due to it being broken and wayland is not available.

**Comment by justchokingaround** [comment]
i tried both iterm2 and sixel, and neither worked

**Comment by jstkdng** [comment]
just tested on my end and works fine. The log is located on `$TMPDIR`, post the contents. Are you using apple silicon?

**Comment by justchokingaround** [comment]
i am on apple sillicon, yes

here is the log:

```
 _    _      _                                           
| |  | |    | |                                _     _   
| |  | | ___| |__   ___ _ __ _____   _  __ _ _| |_ _| |_ 
| |  | |/ _ \ '_ \ / _ \ '__|_  / | | |/ _` |_   _|_   _|
| |__| |  __/ |_) |  __/ |   / /| |_| | (_| | |_|   |_|  
 \____/ \___|_.__/ \___|_|  /___|\__,_|\__, |            
                                        __/ |            
                                       |___/     v2.9.1
[2023-08-24 15:10:47.834] [terminal] [info] TERM="xterm-256color", TERM_PROGRAM="WezTerm"
[2023-08-24 15:10:47.834] [terminal] [info] PTY = /dev/ttys004
[2023-08-24 15:10:47.843] [main] [info] Listening for commands on socket /var/folders/2l/tr01_f590k3bv_1trj7k8tmw0000gn/T//ueberzugpp-81440.socket
```

**Comment by jstkdng** [comment]
wezterm huh, could be an issue on their end? Try plain iterm2 and kitty. iterm2 works fine on my vm.

**Comment by justchokingaround** [comment]
uz++ in iterm2 doesn't work for me, with both the sixel output and iterm2 output

**Comment by jstkdng** [comment]
I see it isn't receiving any commands. Have you tried the socket and stdin methods? Without fifo if possible.

**Comment by justchokingaround** [comment]
yes, i tried the `img` and `fzfub` scripts, in both terminals

**Comment by jstkdng** [comment]
ah, then no wonder it's not working. Under macos, the socket and the logfile are located under `$TMPDIR`, not `/tmp`. Both scripts assume the socket is in `/tmp`

**Comment by justchokingaround** [comment]
oh ok, i'll refactor the scripts and test again either later tonight or tomorrow at worst. is this the case for intel macs as well (i assume yes, since it's on an OS level, but just want to be 100% sure)?

**Comment by jstkdng** [comment]
yeah, it is an OS level issue. Same happens on intel macs.

**Comment by justchokingaround** [comment]
using `$TMPDIR` fixed the issue. with the fifo approach, using `/tmp/` worked just fine. i made a little pr that fixes the example scripts: #121
